CONTRIBUTING.md

```diff
@@ -34,6 +34,15 @@ to upgrade and/or update in order to protect their applications.
 
 For sensitive email communications, please use [our PGP key](http://framework.zend.com/zf-security-pgp-key.asc).
 
+## Documentation
+
+Documentation is in [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/),
```

Comment:
> It does, but it renders all GFM syntax

Well, there are a couple GFM features not supported by CommonMark:

 - Strikethrough (`~~example~~`)
 - Tables

I get where you're coming from though.